Makes/ViewController.m

```diff
@@ -0,0 +1,146 @@
+//
+//  ViewController.m
+//  Makes
+//
+//  Created by Mohamed Ragab on 1/8/17.
+//  Copyright © 2017 Mohamed Ragab. All rights reserved.
+//
+
+#import "ViewController.h"
+#import "Make.h"
+#import "AFNetworking.h"
+#import "Masonry.h"
+#import "MakesCollectionViewCell.h"
+#import "MakeDetailViewController.h"
+
+
+
+
+@interface ViewController ()
+@property (nonatomic, strong) NSMutableArray *makes;
+@property (nonatomic,strong) NSCache* imagesCache;
+@end
+
+@implementation ViewController
+
+- (void)loadMakes
```

Comment:
as equivalent to BAZ all networking calls need to be in separate classes (managers) with their equivalent implementation .. and to be instantiated and called in the dedicated view controllers to set the datasource.